### ISSUE TITLE
Unreal bloom post-processing effect

### ISSUE THREAD

**Issue by Russo-creation** [pmndrs/postprocessing#309]
Hi, 

I would like create a scene where bloom effect is really important but with actual bloom https://vanruesc.github.io/postprocessing/public/demo/#bloom it not that effective. What exactly I would like to achive is 
https://codesandbox.io/s/7mfqw?file=/src/index.js:371-386 

On three.js examples we can find https://threejs.org/examples/?q=bloom#webgl_postprocessing_unreal_bloom_selective 

Is it possible to add this effect to this library?

**Comment by vanruesc** [comment]
Hi,

> Is it possible to add this effect to this library?

yes, this is part of the roadmap (mipmap blur) and actually one of the things I'll do first when I get to it. See https://github.com/vanruesc/postprocessing/issues/279#issuecomment-883609187.

**Comment by Polyterative** [comment]
+1 would really improve my results

**Comment by vanruesc** [comment]
Implemented Unreal/Mipmap bloom in `postprocessing@6.28.0`. The effect can be enabled like this: 

```js
const effect = new SelectiveBloomEffect(scene, camera, {
	luminanceThreshold: 0.1,
	luminanceSmoothing: 0.3,
	mipmapBlur: true,
	intensity: 4.0,
	radius: 0.85
});
```
